src/pages/VenuePage/VenuePage.tsx

```diff
@@ -66,12 +150,22 @@ export const VenuePage: React.FC = () => {
 
   // @debt Remove this once we replace currentVenue with currentVenueNG or similar across all descendant components
   useConnectCurrentVenue();
-  const venue = useSelector(currentVenueSelector);
-  const venueRequestStatus = useSelector(isCurrentVenueRequestedSelector);
+  const postVenue = useSelector(currentVenueSelector);
+  const postVenueLoaded = useSelector(isCurrentVenueRequestedSelector);
```

Comment:
Why are these renamed? There's no point having both of this and the 'fetch via backend function' code. I suggest we just stick to this code as we already had it if it's fast enough now that the excess data fetching has been slimmed down.

Reply:
For testing purposes. Performance tests back up claim we don't need new way of fetching. Restored original logic and naming with 0da80c02d45c99456fdaff937409dc563ca3fdb4